src/pages/Home/components/HomeSlideBanner/HomeSildeBanner.tsx

```
@@ -50,46 +44,44 @@ const HomeSildeBanner = () => {
-          <img
-            key={index}
-            className={styles.BannerImg}
-            src={item.img}
-            alt={item.title}
-          />
+        {homeBannerList.map((item) => (
+          <button type="button" onClick={() => handleToBannerLink(item.link)}>
+            <img
+              key={item.id}
+              className={styles.BannerImg}
+              src={item.img}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)